### PR TITLE
docs: updated polkit/systemd information

### PR DESCRIPTION
**Pull request by kayrus**
/cc @jonboulle @joshix @robszumski

### CODE REVIEW

File: Documentation/architecture.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You should avoid public access to etcd and instead run fleet [from your local la
 
 ## Securing fleetd
 
-It is also recommended to run fleetd under separate `fleet` user and group, and set the permissions of the fleetd API's listening Unix socket to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. Since the fleet daemon uses [D-Bus][d-bus] to communicate with systemd it is necessary to create a [`polkit(8)`][polkit] rule to allow fleetd to communicate with systemd:
+Starting from v216 systemd supports [`polkit(8)`][polkit] rules to verify access for unprivileged users. Thus it is recommended to run fleetd under separate `fleet` user and group, and set the permissions of the fleetd API's listening Unix socket to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. The polkit rule below grants `fleetd` daemon ran by unprivileged `fleet` user to perform actions with systemd through [D-Bus][d-bus]:
```
Comment (jonboulle) [inline Documentation/architecture.md]: Do they need to be the fleet user, or just in the fleet group? It's a little ambiguous from the current wording.

File: Documentation/architecture.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ You should avoid public access to etcd and instead run fleet [from your local la
 
 ## Securing fleetd
 
-It is also recommended to run fleetd under separate `fleet` user and group, and set the permissions of the fleetd API's listening Unix socket to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. Since the fleet daemon uses [D-Bus][d-bus] to communicate with systemd it is necessary to create a [`polkit(8)`][polkit] rule to allow fleetd to communicate with systemd:
+Starting from v216 systemd supports [`polkit(8)`][polkit] rules to verify access for unprivileged users. Thus it is recommended to run fleetd under separate `fleet` user and group, and set the permissions of the fleetd API's listening Unix socket to `0660`. This will require the local user to be in the `fleet` group to perform an action with fleetd. The polkit rule below grants the `fleetd` daemon run by the unprivileged `fleet` user to perform actions with systemd through [D-Bus][d-bus]:
```
Comment (joshix) [inline Documentation/architecture.md]: systemd version 216 or later supports [`polkit(8)`][polkit] rules to control access for unprivileged users. It is recommended to run fleetd under its own `fleet` user and group, and to set the permissions of the fleetd API socket to mode `0660`, allowing only that user and group to write to the socket. This configuration will require a login user to be in the `fleet` group to perform actions with fleetd. The polkit rule below grants the the fleetd process running as the unprivileged `fleet` user to communicate with systemd over [D-Bus][d-bus]:

Comment (kayrus) [inline Documentation/architecture.md]: @joshix awesome. Thanks!

**Comment by jonboulle** [comment]
thanks, lgtm

**Pull request merged**